mllib/src/test/scala/org/apache/spark/ml/classification/NaiveBayesSuite.scala

```diff
@@ -46,6 +51,44 @@ class NaiveBayesSuite extends SparkFunSuite with MLlibTestSparkContext {
     assert(model.theta.map(math.exp) ~== thetaData.map(math.exp) absTol 0.05, "theta mismatch")
   }
 
+  def expectedMultinomialProbabilities(model: NaiveBayesModel, feature: Vector): Vector = {
```

Comment:
Like above, the ml.NaiveBayesModel parameters are all based on Vector and Matrix which is different from the old one, so I just want to make some facility test functions on this kinds of model rather than converting it to old style model.